### PR TITLE
fix: Proper case for Prettier, JavaScript, TypeScript, GitHub, Less

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: docs/rationale.md [hunk 1]
```diff
@@ -34,47 +34,22 @@ Prettier outputs the minimum number of parentheses required to ensure that the b
...
-
-
-### JSX
-
-
-### Boolean expressions
-
-
-### String concatenation
--->
```
Comment (lipis) [inline docs/rationale.md]: I'm guessing that is not needed at the moment..

File: tests/markdown/__snapshots__/jsfmt.spec.js.snap [hunk 2]
```diff
@@ -3861,7 +3861,7 @@ All of JSX and Flow syntax is supported. In fact, the test suite in \`tests/flow
 _is_ the entire Flow test suite and they all pass.
 
 Prettier also supports [TypeScript](https://www.typescriptlang.org/), CSS,
-[LESS](http://lesscss.org/), [SCSS](http://sass-lang.com),
```
Comment (ikatyang) [inline tests/markdown/__snapshots__/jsfmt.spec.js.snap]: We don't need to update test cases/snapshots. 😂

Comment (lipis) [reply tests/markdown/__snapshots__/jsfmt.spec.js.snap]: But there is no harm in it.. right?

Comment (ikatyang) [reply tests/markdown/__snapshots__/jsfmt.spec.js.snap]: Yeah, just 😂.

**Pull request merged**